scripts/lib/sphinx/generateToc.ts

```diff
@@ -126,15 +125,12 @@ export function generateToc(options: {
     tocChildren.push(...orderEntriesByTitle(nestedTocModules));
   }
 
-  tocChildren.push({
-    title: "Release notes",
-    url: pkg.releaseNotesUrl,
-  });
-
-  tocChildren.push({
```

Comment:
For now, we want Qiskit to still have this Changelog link until the release notes folder is implemented.

Also Runtime and Provider no longer need to specify a changelogUrl. Although that's fine to keep it for now while this code is in transition. We only want to remove changelogUrl once possible.

Reply:
Can it temporarily be called "release notes"? Just for laziness before I implement that feature

Reply:
Not a very big deal; but I think you'll simply want the `else` block in the generation script to have the old code?

Reply:
That's true; done!